### DISCUSSION TITLE
Custom install location

### DISCUSSION THREAD

**Discussion by ySergi** [janhq/jan#2660] [Get Help] [answered]
Jan is installed in this location by default: **C:\Users\sergi\AppData\Roaming\Jan**

The problem is that my C drive does not have as much space as the D drive, which is 2TB. I would like to install Jan on the D drive so that it doesn't take up all the remaining space I have on C, which is truly little.

How do I do this?

**Comment by ux-han** [comment, marked as answer]
This problem is solved in https://github.com/janhq/jan/issues/3282
<img width="1110" alt="Screenshot 2024-09-02 at 8 14 33 PM" src="https://github.com/user-attachments/assets/d57fbd91-7059-4178-b131-eb462861b196">

Please try the latest Jan version!
